src/main/java/net/dv8tion/jda/internal/handle/ApplicationCommandPermissionsUpdateHandler.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.dv8tion.jda.internal.handle;
+
+import net.dv8tion.jda.api.events.interaction.command.ApplicationCommandUpdatePermissionsEvent;
+import net.dv8tion.jda.api.utils.data.DataObject;
+import net.dv8tion.jda.internal.JDAImpl;
+import net.dv8tion.jda.internal.entities.GuildImpl;
+
+import java.util.Objects;
+
+public class ApplicationCommandPermissionsUpdateHandler extends SocketHandler
```

Comment:
High chance that this event will not be sent to bots for long, considering the other application command events are dropped as well.

Reply:
The event was added just now, with the new permission system. Why would they remove it now?